com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -223,7 +223,7 @@ Shader "Universal Render Pipeline/Baked Lit"
             #pragma target 4.5
 
             #pragma vertex UniversalVertexMeta
-            #pragma fragment UniversalFragmentMetaBakedLit
+            #pragma fragment UniversalFragmentMetaUnlit
```

Comment:
Not sure why these are changing?

Reply:
Since BakedLit and MetaUnlit did the same thing in their fragment shader, I had BakedLit reuse the Unlit fragment shader. Would you prefer I keep BakedLit separate?